pinot-controller/src/main/resources/webapp/js/init.js

```diff
@@ -98,6 +98,7 @@ var HELPERS = {
 
   executeQuery: function(query, traceEnabled, callback) {
     var url = "/pql?pql=" + query +"&trace=" + traceEnabled;
+    url = encodeURI(url);
```

Comment:
It seemed to work but I did not test the traceEnabled part. But good point, I will encode the query part.